test/smoketest.bats

```diff
@@ -0,0 +1,29 @@
+set -eu
+
+# Vamp installer smoke-test.
+#
+# Uses a fake `kubectl` command
+
+_fake_kubectl=./test-scripts/kubectl
+if [ ! -f "${_fake_kubectl}" ]; then
+  echo "Required support script ${_fake_kubectl} wasn't found. This script is used to fake kubectl in tests to prevent accidental modifying of real Kubernetes clusters, so tests will refuse to run without it."
+  exit 1
+fi
+
+setup() {
+  PATH="./test-scripts/:${PATH}"
+}
+
+@test "a successful install exits zero" {
+  sh install.sh
+  local status=$?
+  [ "$status" -eq 0 ]
```

Comment:
while it works, does it make sense to use bats asserts, e.g. `assert_success` instead?

Reply:
it looks like asserts is a separate package so I guess we can go with the current approach

Reply:
Yes, the separate package was the difficulty. The recommended way to use the `asserts` package is via git submodules which felt like too much to be adding here.